Dockerfile.cicd

```diff
@@ -0,0 +1,36 @@
+FROM golang:1.15-alpine as builder
+
+LABEL maintainer="MinIO Inc <dev@min.io>"
+
+ENV GOPATH /go
+ENV CGO_ENABLED 0
+ENV GO111MODULE on
+
+RUN  \
+  apk add --no-cache git && \
+  git clone https://github.com/minio/minio && cd minio && \
+  git checkout master && go install -v -ldflags "$(go run buildscripts/gen-ldflags.go)"
+
+FROM alpine:3.12
```

Comment:
This looks like duplicated

Reply:
That is a multi-layer build @vadmeste - this avoids go-runtime in the new layer.